src/main/java/org/wildfly/security/digest/SHA512_256MessageDigest.java

```diff
@@ -27,7 +27,7 @@
 /**
  * SHA-512/256 hashing implementation as defined in FIPS PUB 180-4 Secure Hash Standard
  */
-public class Sha512_256MessageDigest extends MessageDigestSpi {
+public class SHA512_256MessageDigest extends MessageDigestSpi {
```

Comment:
Why rename this class?

Reply:
I probably should have put that in it's own commit but it popped up in the middle of this clean up - a while ago we agreed to upper case 3 letter acronyms and camel case 4 or more - this is a new class added in Elytron 1.2 so we can get away with a quick rename before it leaks into any API.